smt/surrogate_models/krg_based.py

```diff
@@ -764,6 +759,85 @@ def _predict_variances(self, x):
         MSE[MSE < 0.0] = 0.0
         return MSE
 
+    def predict_derivatives_variances(self, x):
+        """
+        Give the derivation of the variance of the kriging model (for one input)
+        Parameters:
+        -----------
+        - x: array_like
+        Input
```

Comment:
point where the derivatives are computed. You can add: array_like shape (nx,) or (1, nx) where nx is x dimension.

Reply:
See other doc strings where shape is specified